tests/tests_geomstats/test_riemannian_mean_shift.py

```diff
@@ -69,16 +69,27 @@ def test_single_cluster_riemannian_mean_shift(self):
 
         self.assertAllClose(expected, result)
 
-    @geomstats.tests.np_and_autograd_only
-    def test_double_cluster_riemannian_mean_shift(self):
-        gs.random.seed(10)
-        number_of_samples = 20
-        sphere = Hypersphere(dim=2)
-        metric = HypersphereMetric(2)
-
-        cluster = sphere.random_von_mises_fisher(kappa=20, n_samples=number_of_samples)
+    @staticmethod
+    def _test_double_cluster_init(
+        seed=10,
+        num_of_samples=20,
+        size_of_dim=2,
+        kappa_value=20,
+        orthogonality_of_sphere=3,
+        bandwidth=0.3,
+        tol=1e-4,
+        num_of_centers=2,
+    ):
+        gs.random.seed(seed)
+        number_of_samples = num_of_samples
+        sphere = Hypersphere(size_of_dim)
+        metric = sphere.metric
```

Comment:
btw, maybe we can also make the same change above and do not import `HypersphereMetric`?

Reply:
Done.